frontend/src/components/listofusers/Searchbar.js

```diff
@@ -6,13 +6,13 @@ const Searchbar = ({ setSearchKeyword }) => {
   const { t } = useTranslation()
 
   return (
-    <nav className="navbar-toggler navbar-light bg-light">
+    <nav className="navbar-toggler navbar-light float-right">
       <form className="form-inline">
         <input
-          className="searchbar form-control mr-sm-2"
+          className="form-control mr-sm-2"
           type="search"
-          placeholder={t('search')}
-          aria-label={t('search')}
+          placeholder={t('accepted.search')}
+          aria-label={t('accepted.search')}
```

Comment:
Ez is: accepted.search alapján nem tudnám se fejlesztőként, se fordítóként, hogy ez mi akar lenni.